queries.sql

```diff
@@ -5,4 +5,36 @@ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pickachu';
 SELECT name,escape_attempts FROM animals WHERE weight_Kg > 10.5;
 SELECT * FROM animals WHERE neutered='true';
 SELECT * FROM animals WHERE NOT name='Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Kindly verify that change was made before the `ROLLBACK` query

Reply:
Hello, could you please help me with telling how I can verify that?